AndroidHomeApplication/app/src/main/java/com/example/androidhomeapplication/Utils.kt

```diff
@@ -32,4 +47,11 @@ fun ViewStarsBinding.setRating(activeElements: Int){
     ).forEachIndexed { index, imageView ->
         imageView.setImageActiveState(isActive = (index < activeElements))
     }
-}
\ No newline at end of file
+}
+
+fun Context.readAssetFileToString(fileName: String): String {
+    val stream = this.assets.open(fileName)
+    return stream.bufferedReader().readText()
+}
+
+fun Fragment.getMovieRepository(): MovieRepository = (activity?.application as MovieRepositoryProvider).movieRepository
```

Comment:
it's Kotlin, not Java
use val Fragment.movieRepository: MovieRepository get() =